### ISSUE TITLE
Consider renaming the Catch module?

### ISSUE THREAD

**Issue by iand675** [fused-effects/fused-effects-exceptions#6]
At the moment there's a pure version of `Control.Effect.Catch` in the `fused-effects` package proper, and there's the one in this package. I know some people use the `PackageImports` extension, but I sort of feel that sibling packages shouldn't necessarily force this extension to be enabled. Thoughts?

**Comment by robrix** [comment]
completely agreed 👍

**Comment by patrickt** [comment]
I agree. In fact, I think we should detonate both the `Catch` and `Resource` effects, since we're going to use the lifted versions that @robrix defined in https://github.com/fused-effects/fused-effects/pull/306, rather than define these effects piecemeal.

**Comment by patrickt** [comment]
This will be addressed with https://github.com/fused-effects/fused-effects-exceptions/pull/8
